provider/datagen/src/transform/cldr/cldr_serde/relativetime.rs

```diff
@@ -0,0 +1,146 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Nit: name the file `date_fields.rs`. If other transformers in the future need to pull data from dateFields.json, they should be able to use the same code.

Reply:
Done.